src/main/java/com/eppo/sdk/helpers/CacheHelper.java

```diff
@@ -1,50 +1,35 @@
 package com.eppo.sdk.helpers;
 
-import com.eppo.sdk.constants.Constants;
-import com.eppo.sdk.dto.BanditParameters;
-import com.eppo.sdk.dto.ExperimentConfiguration;
+import cloud.eppo.rac.Constants;
+import cloud.eppo.rac.dto.BanditParameters;
+import cloud.eppo.rac.dto.ExperimentConfiguration;
 import org.ehcache.Cache;
 import org.ehcache.CacheManager;
 import org.ehcache.config.builders.CacheConfigurationBuilder;
 import org.ehcache.config.builders.CacheManagerBuilder;
 import org.ehcache.config.builders.ResourcePoolsBuilder;
 
-/**
- * CacheHelper class
- */
+/** CacheHelper class */
```

Comment:
Assignment caching (done to prevent requesting the customer's logger log duplicate assignments) should also end up in the common code